src/components/layout/CyVerseAppBar.js

```diff
@@ -528,6 +537,23 @@ function CyverseAppBar(props) {
                 toggleDrawer={toggleDrawer}
                 open={open}
             />
+            {open && (
+                <Tooltip title={t("tools")} placement="right" arrow>
+                    <ListItem
+                        button
+                        id={build(ids.DRAWER_MENU, ids.TOOLS_MI)}
+                        className={clsx(classes.nested, classes.listItem)}
+                        onClick={() =>
+                            router.push("/" + NavigationConstants.TOOLS)
+                        }
+                    >
+                        <ListItemIcon>
+                            <LabelImportantIcon className={classes.icon} />
+                        </ListItemIcon>
+                        {open && <ListItemText>{t("tools")}</ListItemText>}
```

Comment:
The `open` boolean is already checked above for the wrapping `ListItem`, so this check isn't required:
```suggestion
                        <ListItemText>{t("tools")}</ListItemText>
```